flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connector-kafka/pom.xml

```diff
@@ -29,6 +29,7 @@ limitations under the License.
     <artifactId>flink-cdc-pipeline-connector-kafka</artifactId>
 
     <properties>
+        <version.kafka>3.2.0</version.kafka>
```

Comment:
3.3 or 3.4 will be better as flink-kafka-connector release-3.3 clearly support 1.19 & 1.20 and release-3.4 bump to Flink 1.20.

Refer to https://mvnrepository.com/artifact/org.apache.flink/flink-connector-kafka.